code/game/objects/items/armor_polish.dm

```diff
@@ -0,0 +1,40 @@
+/obj/item/armorpolish
+	name = "armor polish"
+	desc = "Some canned tuna... oh wait. That's armor polish."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "armor_polish"
+	w_class = WEIGHT_CLASS_TINY
+	var/remaining_uses = 2
+
+/obj/item/armorpolish/examine(mob/user)
+	. = ..()
+	if(remaining_uses != -1)
+		. += "It has [remaining_uses] uses left."
+
+obj/item/armorpolish/afterattack(atom/target, mob/user, proximity)
+	if(istype(target, /obj/item/clothing/suit) || istype(target, /obj/item/clothing/head))
+		var/obj/item/clothing/I = target;
+		//theos said 30/30/20/25
+		//make sure it's not too strong already ((busted))
+		if((I.armor.melee <= 30) || (I.armor.bullet <= 30) || (I.armor.laser <= 20) || (I.armor.energy <= 25))
```

Comment:
You want strictly lesser than here, not lesser or equal as if you have 30/30/20/25, you will still use up the polish but without having any effects

Reply:
This has not been resolved